lib/passport-wsfed-saml2/utils.js

```diff
@@ -4,6 +4,20 @@ var SamlAssertionParserError = require('./errors/SamlAssertionParserError');
 var SamlResponseParserError = require('./errors/SamlResponseParserError');
 var WSFederationResultParserError = require('./errors/WSFederationResultParserError');
 
+exports.certToPEM = function (cert) {
+  cert = cert.match(/.{1,64}/g).join('\n');
```

Comment:
I know the original version modifies the param passed in, but can we change the approach now that it is extracted?

Maybe even something like

```js
const CERT_START = "-----BEGIN CERTIFICATE-----\n";
const CERT_END = "\n-----END CERTIFICATE-----\n";
const certToPEM = (cert) => CERT_START + cert.match(/.{1,64}/g).join('\n') + CERT_END;
```
or 

```js
const CERT_START = "-----BEGIN CERTIFICATE-----";
const CERT_END = "-----END CERTIFICATE-----";
const certToPEM = (cert) => [].concat(
  CERT_START,
  cert.match(/.{1,64}/g).join('\n')
  CERT_END,
  '' // newline at end
).join("\n");
```